### PR TITLE
Cynic Ruins The Game Part 3: Abandoned Crates Modpack

### PR DESCRIPTION
**Pull request by cynic716**
## About The Pull Request
AFTER MUCH DEBATE THIS IS THE FINAL LIST https://pastebin.com/zbdyfryw

## Why It's Good For The Game
E

## Changelog
:cl:
tweak: abandoned crates
code: look it's just a big loot table change check the pastebin also I had to define a couple reagent bottles but the compiler didn't scream about it so it should be fine.
/:cl:

### CODE REVIEW

**Comment by KomradeSpectre** [comment]
>if(67 to 68) new obj/item/reagent_containers/glass/bottle/catmutation(src)

I don't like where this is going.

**Comment by Fikou** [comment]
image
oi mate you got your species loicense?

File: code/modules/mining/abandoned_crates.dm [hunk 1]
```diff
@@ -24,42 +24,41 @@
...
+			new /obj/item/clothing/mask/cigarette/rollie/trippy(src)
 		if(6 to 10)
-			new /obj/item/bedsheet(src)
-			new /obj/item/kitchen/knife(src)
-			new /obj/item/wirecutters(src)
-			new /obj/item/screwdriver(src)
-			new /obj/item/weldingtool(src)
-			new /obj/item/hatchet(src)
-			new /obj/item/crowbar(src)
+			new /obj/item/melee/skateboard/pro(src)
```
Comment (Fikou) [inline code/modules/mining/abandoned_crates.dm]: why remove the improv surgery kit

**Comment by nervere** [comment]
Bwoink mines are a bit too OOC, don't you think? Banhammer too but at least that one is sort of a well-accepted joke.

**Comment by KomradeSpectre** [comment]
```
        if(67 to 68)
            new /obj/item/reagent_containers/glass/bottle/catmutation(src)
        if(69 to 70)
            new /obj/item/stack/ore/bluespace_crystal(src, 5)
```
I think these two are in the wrong order to be fair.

**Comment by Fikou** [comment]
> Bwoink mines are a bit too OOC, don't you think? Banhammer too but at least that one is sort of a well-accepted joke.

i may be wrong but im pretty sure bwoink mines are anchored too or something so you wouldnt be able to move them after opening

**Comment by nervere** [comment]
Fikou is correct.
https://github.com/tgstation/tgstation/blob/5e2623d1779da6043b6ee34f750139192f86a824/code/game/objects/effects/mines.dm#L5

File: code/modules/mining/abandoned_crates.dm (hunk 1 above)
Comment (spessbro) [reply code/modules/mining/abandoned_crates.dm]: cause it's useless probably

**Review by spessbro — APPROVED** [review]

File: code/modules/mining/abandoned_crates.dm (hunk 1 above)
Comment (cynic716) [reply code/modules/mining/abandoned_crates.dm]: it is entirely worthless and everyone knows it, besides it's not like you're gonna need it in fuckin' space

**Comment by cynic716** [comment]
if this doesn't get merged I'm going to oranges's house and licking his toes

File: code/modules/mining/abandoned_crates.dm [hunk 2]
```diff
@@ -74,45 +73,41 @@
...
-				new newcoin(src)
+			new /obj/item/clothing/shoes/kindleKicks(src)
 		if(65 to 66)
 			new /obj/item/clothing/suit/ianshirt(src)
 			new /obj/item/clothing/suit/hooded/ian_costume(src)
 		if(67 to 68)
-			for(var/i in 1 to rand(4, 7))
-				var/newitem = pick(subtypesof(/obj/item/stock_parts) - /obj/item/stock_parts/subspace)
-				new newitem(src)
+			new /obj/item/reagent_containers/glass/bottle/catmutation(src)
```
Comment (plapatin) [inline code/modules/mining/abandoned_crates.dm]: [Worried laughter.]

**Comment by knacker48** [comment]
So before you either got shorts or a syndie pistol. Now you get a skateboard or.... cat mutation toxin?

**Comment by cynic716** [comment]
YEP

**Comment by cynic716** [comment]
for those wondering catmutation=felinid not literal simple mob cat

File: code/modules/mining/abandoned_crates.dm (hunk 2 above)
Comment (Qustinnus) [reply code/modules/mining/abandoned_crates.dm]: This changes the availability of races, no?

**Comment by Shadowflame909** [comment]
I am going to destroy the felind mutation toxin vial

This is not a good meme gamer you've just made a controversial PR

Might I recommend the wearable cat-ears clothing item instead

File: code/modules/reagents/reagent_containers/bottle.dm [hunk 3]
```diff
@@ -309,6 +309,41 @@
...
+
+/obj/item/reagent_containers/glass/bottle/mothmutation
+	name = "ancient bottle"
+	desc = "An old bottle with a faded label from eras gone-by. You can't tell if the label has worn away natrually or if it has been hastily scratched off."
+	list_reagents = list(/datum/reagent/mutationtoxin/moth = 5)
+
+/obj/item/reagent_containers/glass/bottle/mothmutation
+	name = "ancient bottle"
+	desc = "An old bottle with a faded label from eras gone-by. You can't tell if the label has worn away natrually or if it has been hastily scratched off."
+	list_reagents = list(/datum/reagent/mutationtoxin/moth = 5)
```
Comment (willox) [inline code/modules/reagents/reagent_containers/bottle.dm]: you did this one twice

File: code/modules/mining/abandoned_crates.dm (hunk 2 above)
Comment (81Denton) [inline code/modules/mining/abandoned_crates.dm]: Oi m8 you don't have a species availability change loicense
remove this one and the others

File: code/modules/reagents/reagent_containers/bottle.dm [hunk 4]
```diff
@@ -309,6 +309,41 @@
 	desc = "A small bottle containing Bio Virus Antidote Kit."
 	list_reagents = list(/datum/reagent/vaccine/fungal_tb = 30)
 
+/obj/item/reagent_containers/glass/bottle/flymutation
```
Comment (81Denton) [inline code/modules/reagents/reagent_containers/bottle.dm]: Same with all of these

**Review by 81Denton — CHANGES_REQUESTED** [review]

**Comment by cynic716** [comment]
can someone explain how this is even remotely controversial. it's dumb meme items that are absurdly rare drops to begin with

**Comment by Shadowflame909** [comment]
The same way cere station was controversial.

Forced Purrbation or anything like that cause pain that is out of the scope of this game. The same type of pain you'd get from ingesting skewium.

It's traumatic and leads to the growth of an ailment called "Anti-Catgirl players"

If you would not like to see this ailment spread and tear our community apart. I recommend you do not force the mantle of "Cat-player" upon thy playerbase.

As only a few players are well equipped to handle that pain.

If you or a loved one is suffering from forced cat transformation. I recommend you stop and 1. Commit suicide to prevent the effects of the gene growing worse.

2. Just log out of the game.

3. Quickly remove this ailment.

If not some common side-effects are, salt, grief, burn-out, long forum posts, and at its worst point? A rule 0 violation.

Be kind to your fellow space-men. And maybe we all might enjoy playing with each other.

I doubt this though.

**Comment by 81Denton** [comment]
>can someone explain how this is even remotely controversial. it's dumb meme items that are absurdly rare drops to begin with

Race change items/features are very controversial and tend to shit up our repo with pointless, angry arguing. This is why the rule was added in the first place, it's just not worth the drama.

File: code/modules/reagents/reagent_containers/bottle.dm (hunk 3 above)
Comment (spessbandit) [reply code/modules/reagents/reagent_containers/bottle.dm]: natrually

**Comment by optimumtact** [comment]
It's controversial because people who don't like the felinid species do not enjoy being force changed into it.

Just drop the catmutation toxin, it's not a can of worms worth touching until the change freeze on felinids has expired.

**Comment by cynic716** [comment]
THERE EVERYTHING IS FIXED FOREVER

File: code/modules/mining/abandoned_crates.dm [hunk 5]
```diff
@@ -78,23 +79,23 @@
...
-			new /obj/item/reagent_containers/glass/bottle/catmutation(src)
+			new /obj/item/toy/plush/awakenedplushie(src)
 		if(69 to 70)
 			new /obj/item/stack/ore/bluespace_crystal(src, 5)
 		if(71 to 72)
-			new /obj/item/reagent_containers/glass/bottle/lizardmutation(src)
+			new /obj/item/toy/plush/snakeplushie(src)
 		if(73 to 74)
-			new /obj/item/reagent_containers/glass/bottle/mothmutation(src)
+			new /mob/living/simple_animal/pet/gondola(src)
```
Comment (Cyprex) [inline code/modules/mining/abandoned_crates.dm]: gondolas can be killed and the meat can be ground for gondola mutation toxin I believe.

File: code/modules/reagents/reagent_containers/bottle.dm [hunk 6]
```diff
@@ -309,41 +309,6 @@
...
-	name = "ancient bottle"
-	desc = "An old bottle with a faded label from eras gone-by. You can't tell if the label has worn away natrually or if it has been hastily scratched off."
-	list_reagents = list(/datum/reagent/mutationtoxin/felinid = 5)
-
-/obj/item/reagent_containers/glass/bottle/skeletonmutation
-	name = "ancient bottle"
-	desc = "An old bottle with a faded label from eras gone-by. You can't tell if the label has worn away natrually or if it has been hastily scratched off."
-	list_reagents = list(/datum/reagent/mutationtoxin/skeleton = 5)
-
-/obj/item/reagent_containers/glass/bottle/lizardmutation
```
Comment (Cyprex) [inline code/modules/reagents/reagent_containers/bottle.dm]: This is already available via xenobio, skeleton mutation toxin was probably fine as well.

File: code/modules/mining/abandoned_crates.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/mining/abandoned_crates.dm]: if it's not code comments don't use the review feature for it reeeeeeeeeeeeee

File: code/modules/mining/abandoned_crates.dm [hunk 7]
```diff
@@ -228,6 +219,8 @@
 	if(locked)
 		boom(user)
 	else
+		if (qdel_on_open)
```
Comment (Qustinnus) [inline code/modules/mining/abandoned_crates.dm]: there's no need for an else if here. remove it.

Comment (Qustinnus) [reply code/modules/mining/abandoned_crates.dm]: actually no just make it a proper else if instead.

File: code/modules/mining/abandoned_crates.dm [hunk 8]
```diff
@@ -24,42 +25,41 @@
...
-			new /obj/item/melee/baton(src)
+			new /obj/item/toy/snappop/phoenix(src)
+			new /obj/item/toy/snappop/phoenix(src)
+			new /obj/item/toy/snappop/phoenix(src)
+			new /obj/item/toy/snappop/phoenix(src)
+			new /obj/item/toy/snappop/phoenix(src)
 		if(41 to 45)
-			new /obj/item/clothing/under/shorts/red(src)
-			new /obj/item/clothing/under/shorts/blue(src)
+			new /obj/item/reagent_containers/food/snacks/burger/roburger(src)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mining/abandoned_crates.dm]: this burger actually has robot transmutation which basically is a disease that force-borgs you.

File: code/modules/mining/abandoned_crates.dm [hunk 9]
```diff
@@ -9,6 +9,7 @@
 	var/lastattempt = null
 	var/attempts = 10
 	var/codelen = 4
+	var/qdel_on_open
```
Comment (Qustinnus) [inline code/modules/mining/abandoned_crates.dm]: = FALSE

File: code/modules/reagents/reagent_containers/bottle.dm [hunk 10]
```diff
@@ -410,3 +410,4 @@
 /obj/item/reagent_containers/glass/bottle/bromine
 	name = "bromine bottle"
 	list_reagents = list(/datum/reagent/bromine = 30)
+
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/reagent_containers/bottle.dm]: bruh moment

Comment (Qustinnus) [reply code/modules/reagents/reagent_containers/bottle.dm]: bromine moment!!

**Comment by Shadowflame909** [comment]
The Roburger and mutation toxins just seem to be xenobio stuff

Might I recommend other forsaken admin-only items like dueling pistols? Since they cannot be used outside of an actual duel that's consented to.

Also the flash-dark

There's also the useless memorizer. Which is a flash that's been re-iconed to look like a MIB mindwiping device.

**Comment by cynic716** [comment]
I HAVE MADE THE FINAL CHANGE

File: code/modules/mining/abandoned_crates.dm [hunk 11]
```diff
@@ -74,81 +74,72 @@
...
-			new /obj/item/clothing/suit/suspenders(src)
-			new /obj/item/toy/crayon/mime(src)
-			new /obj/item/reagent_containers/food/drinks/bottle/bottleofnothing(src)
+			new /obj/item/toy/plush/nukeplushie(src)
 		if(96)
-			new /obj/item/hand_tele(src)
+			new /obj/item/banhammer(src)
+			new /obj/effect/mine/sound/bwoink(src)
+			new /obj/effect/mine/sound/bwoink(src)
+			new /obj/effect/mine/sound/bwoink(src)
```
Comment (Fikou) [inline code/modules/mining/abandoned_crates.dm]: you still didnt remove the bwoink mines

**Comment by optimumtact** [comment]
enough backseat maintaining

**Comment by optimumtact** [comment]
@cynic716 you need to get that FALSE define in

**Comment by cynic716** [comment]
IT IS DONE

**Pull request merged**